harness/determined/common/experimental/determined.py

```diff
@@ -155,12 +155,15 @@ def create_model(
             description (string, optional): A description of the model.
             metadata (dict, optional): Dictionary of metadata to add to the model.
         """
-        r = self._session.post(
-            "/api/v1/models",
-            json={"description": description, "metadata": metadata, "name": name, "labels": labels},
+
+        # TODO: add notes param to create_model()
```

Comment:
this is out of scope for this PR?

Reply:
no public api changes in this pr, migrating all to the generated bindings is plenty.